### ISSUE TITLE
Limit min/max

### ISSUE THREAD

**Issue by capobecchino** [JustB/placequotes#4]
Set the min/max limit

**Comment by jubstuff** [comment]
Are you talking about min/max width/height for images?

**Comment by capobecchino** [comment]
yes, the min & max for the width & height because if i request a banner with 10000x10000px your server ... KABOOOOMMMMMMMM!

**Comment by jubstuff** [comment]
Ok. The limits are 150x150 to 2000x2000.
